src/main.rs

```diff
@@ -74,6 +74,14 @@ enum Commands {
 		#[clap(default_value = "-")]
 		text: MaybeStdin<String>,
 	},
+	// Count a given charchter
+	CountChar {
+		#[clap(default_value = "-")]
+		text: MaybeStdin<String>,
+
+		#[clap(default_value = " ")]
```

Comment:
Maybe the default value is `""`, and it counts just the overall number of characters in the string?

Reply:
> error: invalid value '' for '[CHARACTER]': cannot parse char from empty string

Reply:
Yeah yeah I know but I was thinking we could add a check for `char == ""` and then just count the chars instead.

Reply:
> Yeah yeah I know but I was thinking we could add a check for `char == ""` and then just count the chars instead.

Used an optional instead.